functions/pipeline/shared/db_access/models.py

```diff
@@ -84,7 +91,7 @@ def __init__(self, training_id, image_id, classification_id, x_min, x_max, y_min
         self.box_confidence = box_confidence
         self.image_confidence = image_confidence
 
-    def convert_to_relative(self):
+    def convert_to_absolute(self):
```

Comment:
@samiyaakhtar So this method and the convert_to_relative on the Tag object are manipulating the current object, not returning a new object. I think returning a new object might be safer and less confusing for consumers of the object. What do you think?

Reply:
Maybe, though it's considered good practice manipulating objects within the class itself too, if that makes readability better I will convert them to static.

Reply:
the more I think about it, I think if it's a static function that sends a new object back it's as good as not using the class object itself. Let me know if you think I should still go ahead and make it static within the Tag and PredictionLabel class